sdks/java/io/parquet/build.gradle

```diff
@@ -25,7 +25,7 @@ applyJavaNature(
 description = "Apache Beam :: SDKs :: Java :: IO :: Parquet"
 ext.summary = "IO to read and write on Parquet storage format."
 
-def parquet_version = "1.10.0"
+def parquet_version = "1.11.1"
```

Comment:
This one you cannot do because of transitive dependency with Avro 1.9 BEAM-8388 for more details.